kr/modules/exporter.md

```diff
@@ -0,0 +1,15 @@
+# Exporter and Reporter
+
+
+## Exporter
+The exporter is a REST API that exports the data of the different databases of the bridgehead in a set of tables.
+It can accept different output formats as CSV, Excel, JSON or XML. It can also export data into Opal.
```

Comment:
```suggestion
It can create different output formats such as CSV, Excel, JSON or XML. It can also export data into [Opal](https://opaldoc.obiba.org/).
```